workspace/src/main/openapi/workspace.yaml

```diff
@@ -1127,6 +1153,8 @@ components:
             defaultRunTemplateDataset:
               runA: "data1"
           datasetCopy: true
+          creationDate: 1649941200000
+          lastUpdate: 1649941200000
```

Comment:
This should be removed and `creationInfo` / `updateInfo` should be added instead